sdk/src/main/java/com/google/cloud/dataflow/sdk/util/BigQueryTableRowIterator.java

```diff
@@ -121,8 +121,20 @@ public static BigQueryTableRowIterator fromTable(TableReference ref, Bigquery cl
 
   /**
    * Constructs a {@code BigQueryTableRowIterator} that reads from the results of executing the
-   * specified query in the specified project.
+   * specified query in the specified project with useLegacySql set to True.
+   *
+   * @deprecated use {@link #fromQuery(String, String, Bigquery, Boolean, Boolean)}.
    */
+  @Deprecated
+  public static BigQueryTableRowIterator fromQuery(
```

Comment:
We can make none backward compatible changes in here.
However, I think users can still choose to use it since it is a public class. (Our integration tests use it)

I think maintaining this deprecated method has very little overheads, and might be worth to have it.
I am okay with either way.